deployment/helm/node-feature-discovery/templates/master.yaml

```diff
@@ -19,8 +21,10 @@ spec:
       labels:
         {{- include "node-feature-discovery.selectorLabels" . | nindent 8 }}
         role: master
+      {{- with .Values.master.annotations }}
       annotations:
-        {{- toYaml .Values.master.annotations | nindent 8 }}
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
```

Comment:
Is there some particular reason to change these? If not, I'd suggest not to change

Reply:
See my explanation in the [comment above](https://github.com/kubernetes-sigs/node-feature-discovery/pull/1331/files#r1317319726)